.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,5 +29,3 @@ Closes <!--- Insert Issue Number(s) this PR addresses. Start by typing # will op
 * [ ] Changes appear not to be breaking changes?
 * [ ] Appropriate automated tests included?
 * [ ] Code style and in-line documentation are appropriate?
-* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
```

Comment:
Is this because we're now adding the milestone?

Reply:
getting rid of unverified